internal/services/gateway/action/run.go

```diff
@@ -145,6 +145,34 @@ func (h *ActionHandler) GetLogs(ctx context.Context, req *GetLogsRequest) (*http
 	return resp, nil
 }
 
+type DeleteLogsRequest struct {
+	RunID  string
+	TaskID string
+	Setup  bool
+	Step   int
+}
+
+func (h *ActionHandler) DeleteLogs(ctx context.Context, req *DeleteLogsRequest) error {
+	runResp, resp, err := h.runserviceClient.GetRun(ctx, req.RunID, nil)
+	if err != nil {
+		return ErrFromRemote(resp, err)
+	}
+	canGetRun, err := h.CanGetRun(ctx, runResp.RunConfig.Group)
```

Comment:
We should avoid everyone able to see the run also delete the logs. Only people that can be able do run actions should be able to delete logs (in future it'll be improved with #13).

So let's use: `h.CanDoRunActions()` instead of `h.CanGetRun()`